### ISSUE TITLE
Allow events to have a priority, specified in hoist.json.  Run events using this priority.

### ISSUE THREAD

**Issue by abutel** [hoist/Hoist#87]

**Comment by bobbynewmark** [comment]
I need this one for UI stuff, as I am checking the Xero Connection by adding an event to the queue.  But if there is a backlog it could take a while to come through, would be great to be able to say this one should be done nextish

**Comment by mejamiewilson** [comment]
@buildmaster Does this work in the hoist.json? Or only in endpoint and hoist-connect?

**Comment by buildmaster** [comment]
No. At the moment it's only API, endpoint and within events with hoist.raise.

## 

Owen Evans
Co-Founder
www.hoist.io(http://www.hoist.io/)
@hoistapps
+1 (415) 712-3680(tel:+1%20(415)%20712-3680)
+64 212014594(tel:+64%20212014594)
